### PR TITLE
[6.x] Update Mitigation by Armor

### PR DESCRIPTION
**Pull request by funjoker**
The needed creature_classlevelstats.basearmor  will be submitted via issue.

Combat Rating ArmorPenetration is removed, because there is no character combat rating Armor Penetration since 4.0.1.

Formula was proofed and calculated in issue
#13818

### CODE REVIEW

File: src/server/game/Entities/Unit/Unit.cpp [hunk 1]
```diff
@@ -1451,39 +1453,23 @@ uint32 Unit::CalcArmorReducedDamage(Unit* victim, const uint32 damage, SpellInfo
...
-        armor -= std::min(armorPen, maxArmorPen);
-    }
-
     if (armor < 0.0f)
         armor = 0.0f;
 
-    float levelModifier = getLevel();
-    if (levelModifier > 59)
-        levelModifier = levelModifier + 4.5f * (levelModifier - 59);
+    uint32 K{};
```
Comment (jackpoz) [inline src/server/game/Entities/Unit/Unit.cpp]: how about using the usual "= 0;" syntax ? it should make things more clear.

Comment (funjoker) [inline src/server/game/Entities/Unit/Unit.cpp]: thanks.

**Comment by funjoker** [comment]
I'm not able to rewrite the system. But take this formula for newer calculation.
I'll provide the BaseArmor list.

**Comment by msoky** [comment]
close?

**Pull request closed without merging**